config/rbac/role.yaml

```diff
@@ -5,52 +5,41 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - events
-  verbs:
-  - create
-  - patch
-- apiGroups:
-  - ""
-  resources:
-  - secrets
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - cert-manager.io
-  resources:
-  - certificaterequests
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - cert-manager.io
-  resources:
-  - certificaterequests/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - sample-issuer.example.com
-  resources:
-  - clusterissuers
-  - issuers
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - sample-issuer.example.com
-  resources:
-  - clusterissuers/status
-  - issuers/status
-  verbs:
-  - get
-  - patch
-  - update
+- apiGroups: [ "" ]
+  resources: [ "events" ]
+  verbs: ["create", "patch"]
+
+- apiGroups: [ "" ]
+  resources: [ "secrets" ]
+  verbs: ["get", "list", "watch"]
+
+- apiGroups: [ "sample-issuer.example.com" ]
+  resources: [ "clusterissuers", "issuers" ]
+  verbs: ["get", "list", "watch"]
+
+- apiGroups: [ "sample-issuer.example.com" ]
+  resources: [ "clusterissuers/status", "issuers/status" ]
+  verbs: [ "patch" ]
+
+
+- apiGroups: [ "cert-manager.io" ]
+  resources: [ "certificaterequests" ]
+  verbs: [ "get", "list", "watch" ]
+
+- apiGroups: [ "cert-manager.io" ]
+  resources: [ "certificaterequests/status" ]
+  verbs: [ "patch" ]
+
+- apiGroups: [ "certificates.k8s.io" ]
+  resources: [ "certificatesigningrequests" ]
+  verbs: [ "get", "list", "watch" ]
+
+- apiGroups: [ "certificates.k8s.io" ]
+  resources: [ "certificatesigningrequests/status" ]
+  verbs: [ "patch" ]
+
+- apiGroups: [ "certificates.k8s.io" ]
+  resources: [ "signers" ]
+  verbs: [ "sign" ]
+  resourceNames:
+  - clusterissuers.sample-issuer.example.com/*
```

Comment:
question: maybe it's because it's early, but I'm surprised that the namespace-scoped issuer doesn't appear here. Is it missing intentionally?

Reply:
This section is for CertificateSigningRequest resources.
Currently, issuer-lib only supports signing those with cluster-scoped issuers since CertificateSigningRequest resources are also cluster-scoped.